tests/VarDumperTest.php

```diff
@@ -31,13 +31,14 @@ public function testExportIncompleteObject(): void
     public function testDumpObject(): void
     {
         $obj = new StdClass();
-        $this->assertEquals("stdClass#1\n(\n)", VarDumper::dumpAsString($obj));
+        $this->assertEquals("stdClass#2\n(\n)", VarDumper::dumpAsString($obj));
```

Comment:
No. It shouldn't modified

Reply:
It must be modified

Reply:
Where did you create the first object?

Reply:
In the first test

Reply:
All test must be independent

Reply:
I agree. Side effects in tests should be avoided.